docs/releasing-files.md

```diff
@@ -163,13 +163,20 @@ Once you have completed this form, the requested outputs will undergo independen
 
 ## How are files reviewed
 
+Before any files are released from the secure server, they are checked by two independent trained OpenSAFELY output checkers. Each output is marked as:
+
+* Approve - output meets disclosure requirements and is safe to be released from the server
+* Approve subject to change - output is an acceptable output type but has outstanding disclosure issues that should be addressed before release from the server
+* Reject - output is no an acceptable output type for release. An example is the release of practice level data which does not meet the [permitted study results policy](https://www.opensafely.org/policies-for-researchers/#permitted-study-results-policy)
```

Comment:
suggestion: a very slight editorial polish here :sparkles:

```suggestion
Before any files are released from the secure server, they are checked independently by two trained OpenSAFELY output checkers. Each checked output is marked as one of the following categories:

* **Approve** — output meets disclosure requirements and is safe to be released
* **Approve subject to change** — output is an acceptable type for release, but has outstanding disclosure issues that must be addressed before release
* **Reject** — output is not an acceptable type for release. An example is the release of practice level data which does not meet the [permitted study results policy](https://www.opensafely.org/policies-for-researchers/#permitted-study-results-policy)
```

Quick summary:

* some small edits for readability
* emphasise the three output review outcomes
* use em-dash instead of hyphen

Reply:
I appeared to have messed up incorporating this suggestion, but pulling all these changes in separately!